swiftmailer/swiftmailer/CVE-2024-28859.yaml

```
@@ -0,0 +1,14 @@
+title:     Deserialization Gadget chain in Swift Mailer
+link:      https://github.com/FriendsOfSymfony1/symfony1/security/advisories/GHSA-wjv8-pxr6-5f4r
+cve:       CVE-2024-28859
+branches:
+    4.x:
+        time:     null
+        versions: ['>=4.0.0']
+    5.x:
+        time:     null
+        versions: ['>=5.0.0']
+    6.x:
+        time:     2020-12-8 19:18:59
+        versions: ['>=6.0.0', '<6.30.0']
+reference: composer://swiftmailer/swiftmailer
```

```suggestion
reference: composer://friendsofsymfony1/swiftmailer
```

(and the filename needs to be changed accordingly)

---

Hello, @xabbuh , thank for you checks. `reference` for the file `swiftmailer/swiftmailer/CVE-2024-28859.yaml` corresponds to `swiftmailer/swiftmailer` repository. But it is archived and there has been a fork. So I suggest maybe making 2 files for referencing both repositories?